### PR TITLE
[RFR] Inject emitter

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
 ```js
 handleSql(['INSERT ... INTO user', userData]);
 ```
+
+## injecting custom eventEmitter
+
+Sg use an eventEmitter internally to handle take and put effect.
```
Comment (djhi) [inline README.md]: s/use/uses

Comment (djhi) [reply README.md]: s/effect/effects/

File: README.md [hunk 2]
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
...
+It is possible to pass your own eventEmitter to sg. This allow to take event from this event emitter.
```
Comment (djhi) [inline README.md]: s/allow/allows/

Comment (djhi) [reply README.md]: s/take event/take events/

**Review by djhi — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
 ```js
 handleSql(['INSERT ... INTO user', userData]);
 ```
+
+## injecting custom eventEmitter
```
Comment (jpetitcolas) [inline README.md]: s/injecting custom eventEmitter/Injecting Custom EventEmitter/

File: README.md [hunk 4]
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
...
+
+Sg use an eventEmitter internally to handle take and put effects.
```
Comment (jpetitcolas) [inline README.md]: s/use/uses/
s/take/`take`/
s/put/`put`/

**Review by djhi — APPROVED** [review]

**Pull request merged**